draft-ietf-quic-transport.md

```diff
@@ -4188,115 +4191,119 @@ ACK Delay:
+{: #ack-range-format title="ACK Ranges"}
 
-Each ACK Block acknowledges a contiguous range of packets by indicating the
+Gap and ACK Range fields use a relative integer encoding for efficiency.  Though
+each encoded value is positive, the values are subtracted, so that each ACK
+Range describes progressively lower-numbered packets.
+
+Each ACK Range acknowledges a contiguous range of packets by indicating the
 number of acknowledged packets that precede the largest packet number in that
-block.  A value of zero indicates that only the largest packet number is
-acknowledged.  Larger ACK Block values indicate a larger range, with
+range.  A value of zero indicates that only the largest packet number is
+acknowledged.  Larger ACK range values indicate a larger range, with
 corresponding lower values for the smallest packet number in the range.  Thus,
-given a largest packet number for the ACK, the smallest value is determined by
+given a largest packet number for the range, the smallest value is determined by
 the formula:
 
 ~~~
-   smallest = largest - ack_block
+   smallest = largest - ack_range
 ~~~
 
-The range of packets that are acknowledged by the ACK Block include the range
-from the smallest packet number to the largest, inclusive.
+The packets that are acknowledged by the ACK Range include the range from the
+smallest packet number to the largest, inclusive.
 
-The largest value for the First ACK Block is determined by the Largest
-Acknowledged field; the largest for Additional ACK Blocks is determined by
-cumulatively subtracting the size of all preceding ACK Blocks and Gaps.
+The largest packet number value for the First ACK Range is determined by the
+Largest Acknowledged field; the largest value for subsequent ACK Range fields is
+determined by cumulatively subtracting the size of all preceding ACK Range and
+Gap fields.
 
-Each Gap indicates a range of packets that are not being acknowledged.  The
-number of packets in the gap is one higher than the encoded value of the Gap
-Field.
+Each Gap field indicates a range of packets that are not being acknowledged.
+The number of packets in the gap is one higher than the encoded value of the Gap
+field.
 
 The value of the Gap field establishes the largest packet number value for the
-ACK Block that follows the gap using the following formula:
+ACK Range that follows the gap using the following formula:
 
 ~~~
   largest = previous_smallest - gap - 2
 ~~~
 
-If the calculated value for largest or smallest packet number for any ACK Block
+If the calculated value for largest or smallest packet number for any ACK Range
 is negative, an endpoint MUST generate a connection error of type
 FRAME_ENCODING_ERROR indicating an error in an ACK frame.
 
-The fields in the ACK Block Section are:
-
-First ACK Block:
-
-: A variable-length integer indicating the number of contiguous packets
-  preceding the Largest Acknowledged that are being acknowledged.
+The fields that form the ACK Ranges field are:
```

Comment:
```suggestion
The fields that form the ACK Ranges are:
```

Reply:
This should be moved above, to before the entire description of how ACK ranges are to be parsed. Probably to the top of {{ack-range-section}}.